src/utils/pgwire/src/pg_server.rs

```diff
@@ -318,121 +330,17 @@ mod tests {
             }
         });
 
-        // explicit parameter (test pre_statement)
```

Comment:
Why are these cases deleted?

Reply:
Becasue for now pgwire is not responsible for parse parameter. So we can't test these case in unit test of pgwire.
And I have moved most of these test in e2e test for extended mode. #8710